server/sonar-web/src/main/js/components/source-viewer/popups/line-actions-popup.js

```diff
@@ -26,7 +25,6 @@ export default Popup.extend({
 
   events: {
     'click .js-get-permalink': 'getPermalink',
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Remove this trailing comma. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3ATrailingComma)